src/cartservice/Dockerfile

```diff
@@ -2,18 +2,17 @@
 FROM mcr.microsoft.com/dotnet/sdk:5.0.103 as builder
 WORKDIR /app
 COPY cartservice.csproj .
-RUN dotnet restore cartservice.csproj -r linux-musl-x64
+RUN dotnet restore cartservice.csproj -r linux-arm64
 COPY . .
-# Fix the issue on Debian 10: https://github.com/dotnet/dotnet-docker/issues/2470
 ENV COMPlus_EnableDiagnostics=0
-RUN dotnet publish cartservice.csproj -p:PublishSingleFile=true -r linux-musl-x64 --self-contained true -p:PublishTrimmed=True -p:TrimMode=Link -c release -o /cartservice --no-restore
+RUN dotnet publish cartservice.csproj -p:PublishSingleFile=true -r linux-arm64 --self-contained true -p:PublishTrimmed=True -p:TrimMode=Link -c release -o /cartservice --no-restore
 
 # https://mcr.microsoft.com/v2/dotnet/runtime-deps/tags/list
-FROM mcr.microsoft.com/dotnet/runtime-deps:5.0.3-alpine3.12-amd64
-RUN GRPC_HEALTH_PROBE_VERSION=v0.3.6 && \
-    wget -qO/bin/grpc_health_probe https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/${GRPC_HEALTH_PROBE_VERSION}/grpc_health_probe-linux-amd64 && \
-    chmod +x /bin/grpc_health_probe
+FROM mcr.microsoft.com/dotnet/runtime-deps:5.0.3-focal-arm64v8
```

Comment:
Does this lose us the ability to run under amd64?

Reply:
If so - I'm wondering if a multi-arch build could be leveraged using `docker buildx`

Reply:
yep, it does.

just putting this PR up in case it's useful to your team. completing this work is out of scope for me atm (per my contract w/ argot).

fwiw, the original app doesn't have ARM support either (#622 in https://github.com/GoogleCloudPlatform/microservices-demo)

Reply:
I guess a bigger fix would be out of scope. I'd suggest then that the same PR be opened against the https://github.com/signadot/microservices-demo repo. We can extend it from there to figure out how to make this run correctly under arm by using this as a reference. Thanks!

Reply:
np, will do!